featuretools/tests/primitive_tests/test_agg_feats.py

```diff
@@ -220,6 +220,11 @@ def test_stack_on_self(es, test_primitive):
 def test_init_and_name(es):
     log = es['log']
 
+    # Add a BooleanNullable column so primitives with that input type get tested
+    boolean_nullable = log.ww['purchased']
+    boolean_nullable = boolean_nullable.ww.set_logical_type('BooleanNullable')
+    log.ww['boolean_nullable'] = boolean_nullable
```

Comment:
I think `log.ww.set_types(logical_types={'purchased':'BooleanNullable'})` would work here as a single-liner if you want (unless this kind of inplace woodwork operation doesn't work as expected... in which case we should look into that)

Reply:
We actually need to have two different columns in the dataframe here, one with `Boolean` logical type and one with `BooleanNullable` in order to test all the primitives with their valid input types. In any case, this test is still having some issues with Dask inputs, and may end up getting changed more, depending on the solution to that.

Reply:
oh, just now seeing that the name of the second column is different!